apps/explorer/src/components/orders/BridgeDetailsTable/index.tsx

```diff
@@ -0,0 +1,318 @@
+  amount?: string | BigNumber | null
+  isLoading?: boolean
+}
+
+// TODO: This is a temporary component to display the bridge amount.
+// It should be replaced with a consolidated component with apps/explorer/src/components/orders/AmountsDisplay
+const BridgeAmountDisplay: React.FC<BridgeAmountDisplayProps> = ({ labelPrefix, bridgeToken, amount, isLoading }) => {
+  if (isLoading) {
+    return <Spinner size="sm" />
+  }
+  if (!bridgeToken) {
+    return (
+      <AmountDetailBlock>
+        <AmountLabel>{labelPrefix}</AmountLabel>
+        <span>N/A</span>
+      </AmountDetailBlock>
+    )
+  }
+
+  const mappedToken = mapBridgeableToErc20(bridgeToken)
+
+  if (amount === undefined || amount === null) {
+    return (
+      <AmountDetailBlock>
+        <AmountLabel>{labelPrefix}</AmountLabel>
+        <span>N/A</span>
+      </AmountDetailBlock>
+    )
+  }
+
+  const formattedAmount = formatSmartMaxPrecision(new BigNumber(amount), mappedToken)
+  const tokenDisplayElement = (
+    <CommonTokenDisplay erc20={mappedToken} network={bridgeToken.chainId as Network} showNetworkName={true} />
+  )
+
+  return (
+    <AmountDetailBlock>
+      <AmountLabel>{labelPrefix}</AmountLabel>
+      <AmountTokenDisplayAndCopyWrapper>
+        <span>{formattedAmount}</span>
+        {isNativeToken(mappedToken.address) ? (
+          tokenDisplayElement
+        ) : (
+          <RowWithCopyButton textToCopy={mappedToken.address} contentsToDisplay={tokenDisplayElement} />
+        )}
+      </AmountTokenDisplayAndCopyWrapper>
+    </AmountDetailBlock>
+  )
+}
+
+const tooltipTextMap = {
+  transactionHash: 'The transaction hash or provider-specific explorer link for the bridge operation.',
+  status: 'The current status of the bridge operation.',
+  amounts: 'The amount of tokens sent to the bridge and the amount received (or expected).',
+  costsAndFees: 'Estimated or actual costs and protocol fees for the bridge operation.',
+  bridgingTime: 'Expected time for the bridge operation to complete.',
+  maxSlippage: 'The maximum allowed slippage for the bridge in percentage.',
+  provider: 'The bridging solution provider.',
+  ownerAddress: 'The account address from which the tokens are bridged.',
+  receiverAddress: 'The account address to which the tokens are bridged on the destination chain.',
+}
+
+const mapBridgeableToErc20 = (
+  bridgeToken: BridgeDetails['source'] | BridgeDetails['destination'],
+): TokenErc20 & { chainId?: Network } => {
+  if (!bridgeToken) {
+    return { address: '', decimals: 18, symbol: 'ERR', name: 'Error Token' } as TokenErc20 & { chainId?: Network }
+  }
+  return {
+    address: bridgeToken.address,
+    decimals: bridgeToken.decimals === undefined ? 18 : bridgeToken.decimals,
+    symbol: bridgeToken.symbol || 'N/A',
+    name: bridgeToken.symbol || bridgeToken.address,
+    chainId: bridgeToken.chainId as Network,
+  }
+}
+
+export type Props = {
+  bridgeDetails?: BridgeDetails
```

Comment:
Why `bridgeDetails` is falsy? It looks like we should not even render `BridgeDetailsTable` if `bridgeDetails` is falsy.